src/deploy/scope_subscription.ts

```diff
@@ -15,7 +15,6 @@ export async function DeploySubscriptionScope(azPath: string, location: string,
         templateLocation ?
             templateLocation.startsWith("http") ? `--template-uri ${templateLocation}`: `--template-file ${templateLocation}`
         : undefined,
-        deploymentMode ? `--mode ${deploymentMode}` : undefined,
```

Comment:
Q: is mode not supported for subscription scope deployments?

Reply:
And if so, and mode is specified by user, please print a warning message specifying that mode will be ignored

Reply:
`az deployment sub create --help` doesn't show a mode parameter.